woocommerce/woocommerce-admin
# Onboarding: Remove plugins step from profiler

Fixes #3894

Removes the plugins step from the profiler and installs plugins directly from the benefits page.  Note that this does not include error handling as this is being handled in https://github.com/woocommerce/woocommerce-admin/pull/3888

### Detailed test instructions:

1. Disable WCS and/or Jetpack.
1. Enable and walk through the profiler until the benefits step. `wp-admin/admin.php?page=wc-admin&step=benefits`
1. Try opting to install plugins and not install them.
1. Make sure plugins are installed and activated correctly and you're redirected to the task list afterwards instead of the previous plugins screen.
